src/components/Home/index.js

```diff
@@ -13,39 +14,49 @@ class Home extends Component {
             loading: true,
             chores: [],
             supplies: [],
-            payments: []
+            payments: [],
+            users: [],
+            home: ""
         };
     }
     componentDidMount() {
         var allChores = []
         var allSupplies = []
         var allPayments = []
+        var allUsers = []
         this.setState({ loading: false });
         this.props.firebase.homes()
             .then(querySnapshot => {
                 querySnapshot.forEach(doc => {
-
-                    doc.data().Chores.forEach(chore => {
+                    var home = doc.data();
+                    home.Chores.forEach(chore => {
                         if (chore.Completed !== true) {
                             allChores.push(chore);
                         }
                     });
-                    doc.data().Supplies.forEach(supply => {
+                    home.Supplies.forEach(supply => {
                         if (supply.Completed !== true) {
                             allSupplies.push(supply);
                         }
                     });
-                    doc.data().Payments.forEach(payment => {
+                    home.Payments.forEach(payment => {
                         if (payment.Completed !== true) {
                             allPayments.push(payment);
                         }
                     })
+                    home.Users.forEach(user => {
+                        allUsers.push(user);
+                    })
                     this.setState({
                         chores: allChores,
                         supplies: allSupplies,
                         payments: allPayments,
+                        users: allUsers,
+                        home: home,
                         loading: false
-                    })
```

Comment:
Commenting on these lines would be appreciated, I see how the idea is to populate the all... arrays, but unsure how it accesses firebase to do so.